posthog/hogql_queries/insights/funnels/funnel_trends.py

```diff
@@ -117,7 +118,7 @@ def _format_single_summary(self, summary):
             data.append(row["conversion_rate"])
             hour_min_sec = " %H:%M:%S" if interval.value == "hour" else ""
             days.append(timestamp.strftime(f"%Y-%m-%d{hour_min_sec}"))
-            labels.append(timestamp.strftime(HUMAN_READABLE_TIMESTAMP_FORMAT))
+            labels.append(format_label_date(timestamp, self._date_range(), self.context.team.week_start_day))
```

Comment:
The constant `HUMAN_READABLE_TIMESTAMP_FORMAT` is now unused and can be removed.